articles/140_topic_and_service_name_mapping.md

```diff
@@ -305,6 +314,24 @@ In ROS 2, topic and service names differ from ROS 1 in that they:
 
 This section lists concerns about the proposed design and alternatives that were considered.
 
+#### Substitution of the Namespace Delimiter
```

Comment:
This subsection should be `###` as the one before it is `##`. (the subsections of this subsection are also affected)